src/main/java/tay/tayApplication.java

```diff
@@ -51,8 +53,12 @@ public void run(final tayConfiguration configuration,
 
         EventResource eventResource = new EventResource();
         environment.jersey().register(eventResource);
-
-
+        
```

Comment:
Not sure what you mean? This is up to date with master at the moment, this was code that you had written for the event stuff.